_data/education.yml

```
@@ -513,6 +513,18 @@ websites:
     regions:
       - se
 
+  - name: Seneca Learning
+    url: https://senecalearning.com
+    img: seneca.svg
+    twitter: SenecaLearning
+    facebook: senecalearning
```

`@senecalearning` seems to be only for India while `@SenecaLearn` is global
```suggestion
    facebook: SenecaLearn
```

---

I changed the Twitter handle, but not the Facebook one, as `@SenecaLearn` doesn't seem to exist for Facebook